qiita_db/metadata_template.py

```diff
@@ -1445,6 +1462,89 @@ def extend(self, md_template):
         # adding the fp to the object
         self.add_filepath(fp)
 
+    def update(self, md_template):
+        r"""Update values in the sample metadata template
```

Comment:
Same here and anywhere else, nobody calls them sample metadata templates, I think everyone calls them sample templates. However this is not a blocking issue, it's more of a nit-pick.